storage-node-v2/src/services/logger.ts

```diff
@@ -0,0 +1,78 @@
+import winston from 'winston'
+import expressWinston from 'express-winston'
+import { Handler } from 'express'
+
+/**
+ * Creates basic Winston logger.
+ *
+ * @returns Winston logger
+ *
+ */
+function createDefaultLogger(): winston.Logger {
+  const levels = {
+    error: 0,
+    warn: 1,
+    info: 2,
+    http: 3,
+    debug: 4,
+  }
+
+  const level = () => {
+    const env = process.env.NODE_ENV || 'development'
+    const isDevelopment = env === 'development'
+    return isDevelopment ? 'debug' : 'warn'
+  }
+
+  const colors = {
+    error: 'red',
+    warn: 'yellow',
+    info: 'green',
+    http: 'magenta',
+    debug: 'white',
+  }
+
+  winston.addColors(colors)
+
+  const format = winston.format.combine(
+    winston.format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss:ms' }),
+    winston.format.colorize({ all: true }),
+    winston.format.printf(
+      (info) => `${info.timestamp} ${info.level}: ${info.message}`
+    )
+  )
+
+  const transports = [new winston.transports.Console()]
+
+  return winston.createLogger({
+    level: level(),
+    levels,
+    format,
+    transports,
+  })
+}
+
+const Logger = createDefaultLogger()
+
+export default Logger
+
+/**
+ * Creates Express-Winston logger handler.
+ *
+ * @returns  Express-Winston logger handler
+ *
+ */
+export function httpLogger(): Handler {
+  const opts: expressWinston.LoggerOptions = {
+    transports: [new winston.transports.Console()],
+    format: winston.format.combine(
+      winston.format.colorize(),
+      winston.format.json()
```

Comment:
This produces an output which is not colorized, but instead json values are broken (ie. `"level":"\u001b[32minfo\u001b[39m"`)

Reply:
Removed incorrect setting.